get

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ARCH=${1:-amd64}
```

Comment:
It would be nice to just default to empty, and use `uname` to detec the correct arch.

```
$ uname -m #x86
x86_64 
$ uname -m #arm64 
aarch64
```

Reply:
I reworked the script to now accept parsed CLI arguments as well as a help output. We now fallback to `uname -m` (converted to amd64/arm64), too. :)

Reply:
Awesome!! :smile: